src/Common/src/System/Security/Cryptography/DerSequenceReader.cs

```diff
@@ -383,6 +383,32 @@ internal string ReadIA5String()
             return TrimTrailingNulls(ia5String);
         }
 
+        internal string ReadT61String()
+        {
+            EatTag(DerTag.T61String);
+            int contentLength = EatLength();
+            string t61String;
+
+            // Technically the T.61 encoding (code page 20261) should be used here, but many
+            // implementations don't follow that and use different character sets. CryptoAPI
+            // on NetFX seems to interpret it as UTF-8 with fallback to ISO 8859-1. OpenSSL
+            // seems to interpret it as ISO 8859-1 with no support for UTF-8.
+            // https://github.com/dotnet/corefx/issues/27466
+            try
+            {
+                var utf8 = (System.Text.Encoding)System.Text.Encoding.UTF8.Clone();
+                utf8.DecoderFallback = System.Text.DecoderFallback.ExceptionFallback;
```

Comment:
This could be simplified to `new Utf8Encoding(false, true)` since it's using the ExceptionFallback.

I'd probably promote this and the ISO-8859-1 encoding objects to statics, no strong opinion on fieldinit or LazyInitializer.EnsureInitialized or just `if (s_utf8WithExceptionsEncoding == null) { s_utf8WithExceptionsEncoding = new Utf8Encoding(false, true); }`

Reply:
I didn't use `new Utf8Encoding(false, true)` the first time because of the additional reference to System.Text.Encoding.Extensions. `LazyInitializer.EnsureInitialized` seems like a reasonable option since it's already used elsewhere in the class.